neqo-crypto/tests/init.rs

```diff
@@ -39,7 +39,7 @@ fn init_nodb() {
 #[cfg(not(nss_nodb))]
 #[test]
 fn init_withdb() {
-    init_db(::test_fixture::NSS_DB_PATH);
+    let _ = init_db(::test_fixture::NSS_DB_PATH);
```

Comment:
```suggestion
    init_db(::test_fixture::NSS_DB_PATH).unwrap();
```
The test should panic here, not later.